lib/modules/dreams_intervention/components/dreams_beneficiary_card.dart

```diff
@@ -145,6 +147,7 @@ class DreamsBeneficiaryCard extends StatelessWidget {
                 canView: canView,
                 isExpanded: isExpanded,
                 onToggleCard: onCardToggle,
+                isAgeBeyondAverage: int.parse(agywDream.age as String) > 24 ? true:false,
```

Comment:
@Kijacode and @ericchingalo I think this condition for knowing if the beneficiary is above 24 should be implemented in the Agyw beneficiary model